covidsafe.proto

```diff
@@ -43,6 +43,11 @@ message GPSCoordinate {
 	double longitude = 2;
 }
 
+message BLTResult {
```

Comment:
Can you please also modify `BLTContactLog` to use `BLTResult` instead of `uuid` only?